docs/pages/reference/api/getting-started.mdx

```diff
@@ -69,31 +75,43 @@ func main() {
 	ctx := context.Background()
 
 	clt, err := client.New(ctx, client.Config{
+		Addrs: []string{
+			"tele.example.com:443",
+			"tele.example.com:3025",
+			"tele.example.com:3024",
+			"tele.example.com:3080",
+ 		},
 		Credentials: []client.Credentials{
 			client.LoadProfile("", ""),
 		},
 	})
+
 	if err != nil {
 		log.Fatalf("failed to create client: %v", err)
 	}
-	defer clt.Close()
 
+	defer clt.Close()
 	resp, err := clt.Ping(ctx)
 	if err != nil {
 		log.Fatalf("failed to ping server: %v", err)
 	}
-	log.Printf("server version: %s", resp.ServerVersion)
+	
+	log.Printf("Example success!")
+	log.Printf("Example server response: %s", resp)
+	log.Printf("Server version: %s", resp.ServerVersion)
 }
+EOF
 ```
 
-Now you can run the program which will connect the client to the Teleport Auth server and fetch the server version.
+Now you can run the program and connect the client to the Teleport Auth server and fetch the server version.
```

Comment:
Note to self:  `and fetch the server version.` to ` to fetch the server version.